collectors/cmdb/hpsm/src/main/java/com/capitalone/dashboard/collector/DefaultHpsmClient.java

```diff
@@ -783,6 +782,8 @@ private void handleChangeSoapMessage(SOAPBodyElement keysTag) throws SOAPExcepti
 		String closedTime = getStringValueFromMap(map,HpsmCollectorConstants.INCIDENT_CLOSE_TIME);
 		if (!StringUtils.isEmpty(closedTime)) {
 			incident.setClosedTime(closedTime);
+		} else {
```

Comment:
(782 - 788) If else can be written as incident.setClosedTime((!StringUtils.isEmpty(closedTime)) ? closedTime : 0L);